src/utils/rest-api.js

```diff
@@ -342,11 +342,13 @@ export function createStudy(
     studyName,
     studyDescription,
     caseName,
+    duplicateCase,
```

Comment:
Rename caseName by caseUuid

Reply:
I renamed and cleaned a bit create-study-dialog too